README.md

```diff
@@ -0,0 +1,163 @@
+ti-auth.js
+==========
+
+`ti-auth.js` is a small wrapper to integrate modules into the [Travel Intelligence Portal](https://travel-intelligence.com) providing:
+- **Authentication** - unauthorized users will be redirected to the TI Portal login page and will return after successful authentication.
+- **User Context** - user information, preferences and grants are made available to the application.
+- **Utility functions** - for signout and reauthorization.
+
+
+## Installation
+via Bower:
+
+    $ bower install git@github.com:travel-intelligence/ti-auth.js --save
+
+or via NPM:
+
+    $ npm install https://github.com/travel-intelligence/ti-auth.js --save-dev
+
+## Quick Start
+
+For ease of use the library exposes one global JavaScript object called **TiAuth**.
+
+After including the library in the in your application’s index.html file only 2 simple steps are needed to use `ti-auth.js` for authentication:
+
+1. **Configuration** - depending on the target environment (production/staging/local etc.) the Travel Intelligence API and the
```

Comment:
Oh, just another note here that it could be confusing to mention the `DASHBOARD_URL` as `https://portal.travel-intelligence.com`. It’s currently still `https://travel-intelligence.com`, right?